api/openapi-spec/swagger.json

```diff
@@ -6789,6 +6789,10 @@
       "description": "NodeSelector is a selector which must be true for the vmi to fit on a node.\nSelector which must match a node's labels for the vmi to be scheduled on that node.\nMore info: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/\n+optional",
       "type": "object"
      },
+     "podAnnotations": {
```

Comment:
hey, i was pinged for another review. My previous comment mostly summarizes my thoughts still. I'd rather we didnt' create another api field to represent pod annotations.

Reply:
Thanks, I wanted to be sure about  the field.